articles/references/modules/toc.yml

```diff
@@ -0,0 +1,6 @@
+- name: Configuration
+  href: configuration.md
+- name: IModSpec File
+  href: imodspec_file.md
+- name: Factory Extensions
+  href: factory_extensions.md
```

Comment:
For maintainability purposes, is it possible for the `name`s to be blank and have DocFX automatically populate it with the article title from an `xref`? This may require that we change TOC files from `.yml` to `.md`.